litentry/core/data-providers/src/discord_litentry.rs

```diff
@@ -88,26 +86,52 @@ impl DiscordLitentryClient {
 	) -> Result<DiscordResponse, Error> {
 		let guild_id_s = vec_to_string(guild_id)?;
 		let handler_s = vec_to_string(handler)?;
-		let path = "/discord/commented/idhubber".to_string();
-		self.client
-			.get_with::<String, DiscordResponse>(
-				path,
-				&[("guildid", &guild_id_s), ("handler", &handler_s)],
-			)
-			.map_err(|e| Error::RequestError(format!("{:?}", e)))
+		let path = "/discord/commented/idhubber".into();
+		let query = vec![("guildid", guild_id_s.as_str()), ("handler", handler_s.as_str())];
+
+		let res = self
+			.client
+			.get_with::<String, DiscordResponse>(path, query.as_slice())
+			.map_err(|e| Error::RequestError(format!("{:?}", e)));
+
+		res
 	}
 }
 
 #[cfg(test)]
 mod tests {
-	use crate::discord_litentry::DiscordLitentryClient;
+	use super::*;
+	use httpmock::prelude::*;
+	use lc_mock_server::standalone_server;
 	use std::vec::Vec;
 
 	#[test]
 	fn check_join_work() {
-		let guildid: u64 = 919848390156767232;
-		let guild_id_vec: Vec<u8> = format!("{}", guildid).as_bytes().to_vec();
-		let handler_vec: Vec<u8> = "againstwar%234779".as_bytes().to_vec();
+		standalone_server();
+		let server = httpmock::MockServer::connect("localhost:9527");
+
+		let body = DiscordResponse {
+			data: true,
+			message: "success".into(),
+			has_errors: false,
+			msg_code: 200,
+			success: true,
+		};
+
+		let path = "/discord/joined";
+		server.mock(|when, then| {
+			when.method(GET)
+				.path(path)
+				.query_param("guildid", "919848390156767232")
+				.query_param("handler", "againstwar#4779");
+			then.status(200).body(serde_json::to_string(&body).unwrap());
+		});
+
+		let guildid = "919848390156767232";
+		// let handler = "againstwar%234779";
```

Comment:
maybe remove this line.

Reply:
we need to move this guildid into configs, and out of codes